examples/sweeps/sweep_test.go

```diff
@@ -0,0 +1,96 @@
+package sweeps
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/moovfinancial/moov-go/pkg/moov"
+)
+
+func Example_SweepConfigs_Endpoints() {
+	mc, err := moov.NewClient()
+	if err != nil {
+		fmt.Printf("new client: %v", err)
+		return
+	}
+
+	var (
+		accountID = "2f29c185-91d8-40c7-8963-3915aa673c21"
+		walletID  = "ce906d91-61ef-47eb-8e15-38f80b863200"
+
+		ctx        = context.Background()
+		minBalance = "100.0"
```

Comment:
I know one decimal is supported, but should we show `100.00` just to suggest the more natural format?

Reply:
Yeah, that makes sense, updating